prerequisites.py

```diff
@@ -210,6 +211,20 @@ def __init__(self):
         # use dispatch pattern to invoke method with same name
         getattr(self, args.command)()
 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/s-maggiolo/cms/file/2192738254/issues/source?bid=3167167&fileBranchId=3167183#l214)